documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,324 @@
+
+When creating an {ocp} to {ocp} migration plan for a VM with 2 NICs, it creates a plan of `Unknown` status, causing the `forklift-controller` to fail. link:https://issues.redhat.com/browse/MTV-704([MTV-704])
+
+.Difficult to capture errors logged in populator pods from customers
+
+When `ovirt-img` fails to download the image, the reason for the failure does not appear in the populator controller or any events log. The pod keeps restarting after each failure, so customer populator pod error logs can only show where the reason for failure if they happen to log exactly when the pod is in error state. link:(https://issues.redhat.com/browse/MTV-725([MTV-725])
+
+.Conversion pod does not use the transfer network
+
+When creating a migration plan using the `transferNetwork`, the `virt-v2v` pod uses the default pod network instead of the transfer network. link:https://issues.redhat.com/browse/MTV-835([MTV-835])
+
+.Creating a host secret requires validating the secret before creating the host
+
+When creating a host secret, you must validate the secret before creating the host. The order for creating a host secret:
+
+. Create a secret
+. Create a host
+. Update the host owner reference
+
+link:https://issues.redhat.com/browse/MTV-868([MTV-868])
+
+.Migrating VMs from RHV to OpenShift Virtualization does not work with an encrypted RBD
+
+Migrating a VM from RHV to OpenShift Virtualization fails during `qemu-img create`, returning a `Cannot grow device files` error. The pod sees a smaller file. link:https://github.com/kubev2v/forklift-console-plugin/issues/852([MTV-852])
+
+.RHV provider `ConnectionTestSucceeded` True response from wrong URL
+
+The `ConnectionTestSucceeded` process produces a `True` result even when the wrong URL is entered. link:https://issues.redhat.com/browse/MTV-740([MTV-740])
+
+.MTV Inventory does not show extended volume of VM disk in RHV
+
+The extended volume of a VM disk does not appear in the MTV Inventory for an RHV provider after it is already connected. The connected RHV provider does not update the volume of the VM disk. link:https://issues.redhat.com/browse/MTV-829([MTV-829])
+
+.`ForkliftController` status shows `Failure` after upgrading from MTV 2.4.2 to MTV 2.5.0
+
+After upgrading MTV from version 2.4.2 to 2.5.0, the `ForkliftController` returns a `Failure` status and an error message. link:https://issues.redhat.com/browse/MTV-702([MTV-702])
+// 
+// .Migration fails when a vSphere Datacenter is nested inside a folder
+// 
+// Migrating a vSphere Datacenter succeeds when it is directly under the `/vcenter`, but fails when it is stores inside a folder. The migration raises an error. link:([MTV-796])
+
+.Fail to migrate a VM with NVME disks from vSphere 
+
+When migrating a VM with NVME disks from vSphere, the migration process fails and the Web Console shows that the `Convert image to kubvirt` stage is 'running` but did not finish successfully. link:https://issues.redhat.com/browse/MTV-963([MTV-963])
```

Comment:
```suggestion
When migrating a VM with NVME disks from vSphere, the migration process fails, and the Web Console shows that the `Convert image to kubvirt` stage is 'running` but has not finished successfully. link:https://issues.redhat.com/browse/MTV-963([MTV-963])
```

You are confusing verb tenses. 
You need to ensure your writing meets basic grammar